src/components/ConfirmationModal.tsx

```diff
@@ -0,0 +1,45 @@
+import styled from "@emotion/styled";
+import Modal, { Variant } from "@leafygreen-ui/confirmation-modal";
+
+interface ConfirmationModalProps {
+  "data-cy"?: string;
+  title: string;
+  open?: boolean;
+  onConfirm?: () => void;
+  onCancel?: () => void;
+  buttonText: string;
+  variant?: Variant;
+  requiredInputText?: string;
+  submitDisabled?: boolean;
+}
+
+export const ConfirmationModal: React.FC<ConfirmationModalProps> = ({
+  buttonText,
+  children,
+  "data-cy": dataCy,
+  onCancel,
+  onConfirm,
+  open,
+  requiredInputText,
+  submitDisabled,
+  title,
+  variant,
+}) => (
+  <StyledModal
+    buttonText={buttonText}
+    data-cy={dataCy}
+    onCancel={onCancel}
+    onConfirm={onConfirm}
+    open={open}
+    requiredInputText={requiredInputText}
+    submitDisabled={submitDisabled}
+    title={title}
+    variant={variant}
+  >
+    {children}
+  </StyledModal>
+);
+
+const StyledModal = styled(Modal)`
```

Comment:
does it make sense to export StyledModal as ConfirmationModal? The props seem to be 1:1.